app/src/androidTest/kotlin/at/bitfire/davdroid/servicedetection/CollectionListRefresherTest.kt

```diff
@@ -719,4 +720,15 @@ class CollectionListRefresherTest {
         }
 
     }
-}
\ No newline at end of file
+}
+
+@Module
+@TestInstallIn(
+    components = [SingletonComponent::class],
+    replaces = [PushRegistrationWorker.PushRegistrationWorkerModule::class]
+)
+abstract class FakePushRegistrationWorkerModule {
+    // Provides empty set of listeners
+    @Multibinds
+    abstract fun defaultOnChangeListeners(): Set<DavCollectionRepository.OnChangeListener>
+}
```

Comment:
This will apply app-wide. So it doesn't belong to a specific test class and should be placed somewhere else.

Probably directly in the`androidTest` root in a `TestModule`, in analogy to the `OseFlavorModule` etc.